libs/ui/src/components/NeoModal/NeoModal.vue

```diff
@@ -31,6 +32,7 @@ const props = withDefaults(
     noShadow?: boolean
     maxHeight?: string | number
     mobileBreakpoint?: string
+    disableTop?: boolean
```

Comment:
not a very clear prop name

Reply:
Please let me know what would be best suited for this. Perhaps just `top`?